src/utils/changelog.ts

```diff
@@ -365,6 +380,7 @@ async function getPRAndMilestoneFromCommit(
       hash.slice(1),
       {
         pr: commit?.associatedPullRequests.nodes[0]?.number ?? null,
+        prBody: commit?.associatedPullRequests.nodes[0]?.body ?? null,
```

Comment:
Kinda WET but meh.

Reply:
Yeah, I'll see if I can tidy up this and the tests in a future PR. Tests also annoy me with the repetition.

Reply:
#298